tests/integration/test_image.py

```diff
@@ -42,6 +47,15 @@ class Commands(NamedTuple):
         name="file permission to /usr/local/bin (create)", command="touch /usr/local/bin/test_file"
     ),
     Commands(name="install microk8s", command="sudo snap install microk8s --classic"),
+    # This is a special helper command to configure dockerhub registry if available.
+    Commands(
+        name="configure dockerhub mirror",
+        command="""echo 'server = "{registry_url}"
+
+[host.{hostname}:{port}]
+capabilities = ["pull", "resolve"]
+' > /var/snap/microk8s/current/args/certs.d/docker.io/hosts.toml""",
+    ),
     Commands(name="wait for microk8s", command="microk8s status --wait-ready"),
```

Comment:
I believe a restart of microk8s is required : https://github.com/charmed-kubernetes/actions-operator/blob/5c6377ed695d52b8a1693f07b7d641e245269123/src/bootstrap/index.ts#L116-L117